### ISSUE TITLE
NetFlow conversion test cases

### ISSUE THREAD

**Issue by SecDorks-Kirc** [CESNET/ipfixcol#121]
The test suite as it is currently present in IPFIXcol only supports IPFIX files as input traffic. An integral part of IPFIXcol is however the code that converts from NetFlow v5 and NetFlow v9 to IPFIX. Hence, non-IPFIX input traffic should be tested and supported by the test suite too.

The most straightforward solution would be to use *tcpreplay* for replaying NetFlow traffic. In case *tcpreplay* is not available on the system, a warning should be printed and the test should be successful.

As a concrete example, I've prepared a test case based on NetFlow traffic for #118.

**Comment by thorgrin** [comment]
I agree that current test cases do not cover the functionality of the collector as well as they should. I'm ready to look into and accept any patches adding more sensible tests. However, I often have problems with tcpreplay on local machine, so I'd need to give your test a try.
